src/Routes/SeasonsController.ts

```diff
@@ -2,21 +2,101 @@ import * as express from "express";
 import { IDataProvider } from "../DataProviders/IDataProvider";
 import { BaseController } from "./BaseController";
 
-/*
- * Seasons controller
+/**
+ * @swagger
+ *
+ * definitions:
+ *   Seasons:
+ *     type: object
```

Comment:
array?

Reply:
Yes.

Reply:
This is the singular object def in the array